custodial-copy-backend/src/main/scala/uk/gov/nationalarchives/custodialcopy/Processor.scala

```diff
@@ -136,7 +136,7 @@ class Processor(
   private def createMetadataFileName(entityTypeShort: String) = s"${entityTypeShort}_Metadata.xml"
 
   private def toCustodialCopyObject(potentialMessage: Option[ReceivedSnsMessage]): IO[List[CustodialCopyObject]] = potentialMessage match {
-    case Some(IoReceivedSnsMessage(ref, _)) =>
+    case Some(IoReceivedSnsMessage(ref, _, _)) =>
```

Comment:
I know this doesn't really affect anything but is it worth naming the `deleted` variable here and passing it into the `fromType` method, just so things match up?